### ISSUE TITLE
ReadyPlayerMe avatars have abnormally large collision hulls.

### ISSUE THREAD

**Issue by Aitolda** [vircadia/vircadia#1537]
This doesn't sound like a big deal at first, but I'm discovering they cant' fit through standard doorways. This is a bigger problem than it seems as most new users will be using these avatars. Not sure if this is something we can automatically fix on import through the app.
<img width="354" alt="hull" src="https://user-images.githubusercontent.com/21160271/146663122-1067fab6-1624-442d-9c32-6393365b25aa.png">

**Comment by Aitolda** [comment]
[Example Avatar](https://aitolda.s3.amazonaws.com/Avatars/ReadyPlayerMeAvatars/ReadyPlayerMeTes_12-16-2021.glb)

**Comment by Aitolda** [comment]
It was suggested to me that the radius of said collider might being handled [here](https://github.com/vircadia/vircadia/blob/cd2d6847ee8a042cc469872a8a8b815c817c080a/libraries/avatars-renderer/src/avatars-renderer/SkeletonModel.cpp#L334)

**Comment by Aitolda** [comment]
One thing I noticed is that this is an A-pose avatar rather than a T-pose.  I'm wondering if perhaps the calculation happens halfway up the model, as most avatars I've used are T-pose and therefore the middle point of the avatar less wide. My "bug" avatar for example![woodyhull](https://user-images.githubusercontent.com/21160271/146665050-fa303c83-2953-4873-9c94-0ed5df4cc1c2.jpg)
![reggiehull](https://user-images.githubusercontent.com/21160271/146665016-f56adfc7-8bb3-4c38-8687-6fc751adc97a.jpg)

The rest post of this is T, and the pill seems closer to his waist size. Woody is similar.

![woodyhull](https://user-images.githubusercontent.com/21160271/146665050-fa303c83-2953-4873-9c94-0ed5df4cc1c2.jpg)

Or perhaps its the radius from the hip joint.
